### DISCUSSION TITLE
Content to a learning module not available

### DISCUSSION THREAD

**Discussion by martapodarta** [community/community#29775] [Other Feature Feedback, Questions, & Ideas]
Hello, 
I cannot access module 7, titled ‘Using Github Script’ in this Github Actions learning path: https://lab.github.com/githubtraining/devops-with-github-actions. Does a content for this module is still available in ‘Learning Lab’? I get page no found when I click on module 7 link: https://lab.github.com/githubtraining/github-actions:-using-github-script

**Comment by samus-aran** [comment]
Hi @martapodarta, GitHub Learning Lab has now been deprecated ( on September 1, 2022). However, we recently launched GitHub Skills, learn more about the shutdown and GitHub Skills [here](https://github.com/orgs/community/discussions/28323).
